packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -149,6 +149,8 @@ export interface CallAdapterCallManagement {
     createStreamView(remoteUserId?: string, options?: VideoStreamOptions): Promise<void | CreateVideoStreamViewResult>;
     disposeStreamView(remoteUserId?: string, options?: VideoStreamOptions): Promise<void>;
     // @beta
+    getEnvironmentInfo(): Promise<EnvironmentInfo>;
```

Comment:
this might have to be optional to prevent a breaking change to this interface

Reply:
Although looking at other new ones we added like `sendDtmfTone` these may also need to be optional

Reply:
Added making these required again to the major bump list: https://skype.visualstudio.com/SPOOL/_wiki/wikis/SPOOL.wiki/31318/What-we-would-like-to-break-when-we-do-a-major-bump
But for now I'd recommend going through newly added adapter interfaces and making them optional